client/package.json

```diff
@@ -13,32 +13,31 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^14.2.4",
-    "@angular/cdk": "^14.2.3",
-    "@angular/common": "^14.2.4",
-    "@angular/compiler": "^14.2.4",
-    "@angular/core": "^14.2.4",
-    "@angular/flex-layout": "^14.0.0-beta.40",
```

Comment:
Nice job getting rid of `flex-layout`.

Reply:
I'm not convinced this is actually ideal. The appearance of the cards and the lists is quite messed up by it. I was *just* about to make a commit that goes back to flex-layout so that we at least have cards that look decent. The list is still very broken either way. I'm not convinced that I have enough time before Lab 3 on *Tuesday* to do anything very good here. @NicMcPhee - Thoughts?